### DISCUSSION TITLE
ArgoCD Notification Controller won't ignore sync history

### DISCUSSION THREAD

**Discussion by sulaimanovm** [argoproj/argo-cd#13182] [Q&A]
Hey folks, 
I have ArgoCD deployed via Helm Charts and configured slack subscription to receive messages about sync status.
However, upon deployment ArgoCD Notification Controller scans through and sends messages for the past sync history thus spamming slack channel.
1.**Is there way to tell it to ignore past history without removing sync history?**
2.**How can I set when condition to only trigger when there is app diff?**
Here is my trigger config:
`- description: Application syncing has succeeded
        send:
        - app-sync-succeeded
        when: app.status.operationState.phase in ['Succeeded']`

**Comment by jaideepr97** [comment]
hi @sulaimanovm 
have you taken a look at https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/triggers/#avoid-sending-same-notification-too-often?
